sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -496,8 +516,7 @@ def _write_for_binary_encoding(
     elif isinstance(value, _Undefined):
         binary_writer.write(bytes([_PartitionKeyComponentType.Undefined]))
 
-
-def _get_partition_key_from_partition_key_definition(
+def get_partition_key_from_definition(
```

Comment:
is there a need for this name change? I believe we purposely had it spelled out so it would be incredibly obvious to anyone reading the code what the method was doing